internal/core/core.go

```diff
@@ -524,3 +524,7 @@ func levenshteinWithMax(s1 []rune, s2 []rune, maxValue float64) float64 {
 func Identity[T any](t T) T {
 	return t
 }
+
+func PtrTo[T any](v T) *T {
```

Comment:
At some level I wish we would just copy and paste this into packages we use it, since `core.PtrTo` is a long word for a simple thing, and it means we keep importing core everywhere which makes modifying it for unrelated reasons cause rebuilds of everything, but, I guess it's fine

Reply:
I mean, everything uses `core` for many other things (why is `TextPos` a named type??) so this doesn’t make material progress toward unwinding the dependency, but sure

Reply:
No, it's alright, just leave it; the PR seems okay as-is.